File: clients/simple.php
<?php
	log_value('localhost', 8464, "hello:1");
	log_value('localhost', 8464, "world:1");

	echo "ok\n";

	function log_value($server, $port, $value){

		$fp = fsockopen("udp://$server", $port, $errno, $errstr);
		if ($fp){
			fwrite($fp, "$value\n");
			fclose($fp);
		}
	}
?>

File: clients/simple.pl
#!/usr/bin/perl

use strict;
use warnings;

use IO::Socket;

&log_value('localhost', 8464, "hello:1");
&log_value('localhost', 8464, "world:1");
print "ok\n";

sub log_value {
	my ($server, $port, $value) = @_;

	my $sock = new IO::Socket::INET(
		PeerAddr => $server,
		PeerPort => $port,
		Proto => 'udp',
	) or die('Could not connect: $!');

	print $sock "$value\n";

	close $sock;
}
